src/isotools/_transcriptome_io.py

```diff
@@ -996,6 +1000,13 @@ def _read_gtf_file(file_name, chromosomes, infer_genes=False, progress_bar=True)
             except ValueError:
                 logger.error('issue with key value pairs from gtf:\n%s', ls[8])
                 raise
+
+            # gtf of transcriptome reconstructed by external tools may include entries without strand, which can't be mapped to the genome, so skip them
+            if ls[6] not in ('+', '-'):
+                logger.warning('skipping line with unknown strand:\n%s', line)
+                skipped[ls[2]].add([i.split(' ')[-1].strip('"') for i in ls[-1].split(sep=';') if f'{ls[2]}_id' in i][0])
```

Comment:
Can you add a comment, what `i.split(' ')[-1].strip('"')` and `f'{ls[2]}_id' in i` are/do?